backend/server/migrations/0010-create-question.js

```diff
@@ -0,0 +1,43 @@
+
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Questions', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER,
+    },
+    text: {
+      type: Sequelize.STRING,
+    },
+    initialImage: {
+      type: Sequelize.STRING,
+    },
+    finalImage: {
+      type: Sequelize.STRING,
+    },
+    initialSound: {
+      type: Sequelize.STRING,
+    },
+    finalSound: {
+      type: Sequelize.STRING,
+    },
+    createdAt: {
+      allowNull: false,
+      type: Sequelize.DATE,
+    },
+    updatedAt: {
+      allowNull: false,
+      type: Sequelize.DATE,
+    },
+    user_id: {
+      type: Sequelize.STRING,
+      references: {
+        model: 'Users',
+        key: 'id',
+        as: 'user_id',
```

Comment:
Each question should belong to an experiment, not a user. Please, fix this, @maverick2000. Thanks.